src/plugins/gs-flatpak.c

```diff
@@ -2333,6 +2333,30 @@ gs_flatpak_refine_metadata_from_installation (GsFlatpak *self,
 	return TRUE;
 }
 
+static void
+ensure_default_branches (GsFlatpak *self)
+{
+	g_autoptr(GPtrArray) xremotes = NULL;
+	guint i;
+
+	xremotes = flatpak_installation_list_remotes (self->installation, NULL, NULL);
+	for (i = 0; i < xremotes->len; i++) {
```

Comment:
You should check for `xremotes != NULL` and return early, since `flatpak_installation_list_remotes()` can fail